sqs_launcher/__init__.py

```diff
@@ -11,56 +11,66 @@ class for running sqs message launcher
 # start imports
 # ================
 
-import boto3
-import boto3.session
 import json
-import os
 import logging
+import os
+
+import boto3
+import boto3.session
 
 # ================
 # start class
 # ================
 
 sqs_logger = logging.getLogger('sqs_listener')
 
+
 class SqsLauncher(object):
 
-    def __init__(self, queue, create_queue=False, visibility_timeout='600'):
+    def __init__(self, queue=None, queue_url=None, create_queue=False, visibility_timeout='600'):
```

Comment:
I don't agree with removing the `create_queue` argument.  I think there are probably  projects which use this library only for sending messages, and don't use the listener component at all.  In such projects not being able to allow the launcher to create a queue could be an obstacle in development.  I'm imaging a microservice-powered ecosystem in which SQS is the primary  means of communication between services, and queues are generated on-the-fly.  Some services might only be broadcasting to particular queues, which are then listened to at some point in the future by another service.
What I'm trying to say is, allowing the launcher to create a queue actually makes asynchronous architecture base on SQS a little simpler.

Reply:
Hi, I totally agree with you, but I haven't removed the `create_queue` argument

Reply:
I'm sorry, you're right.  In the description you wrote 'removed error queue creation...since it is created on launcher initialization" and somehow the two ends of the sentence collided in my brain to form a new thought.  Then, I read the code armed with an incorrect belief that that was one of the changes you made, so I didn't see that it wasn't true...  I think I have to work a little harder on not making any assumptions.

Reply:
It's fine, thanks for letting me contribute